test/unittest/tests/packing/test_intpack.cpp

```diff
@@ -0,0 +1,416 @@
+/*-
+ * Copyright (c) 2014-present MongoDB, Inc.
+ * Copyright (c) 2008-2014 WiredTiger, Inc.
+ *	All rights reserved.
+ *
+ * See the file LICENSE for redistribution information.
+ */
+
+/*
+ * This file unit tests the macros and functions contained in intpack_inline.h
```

Comment:
np: Seems that this comment can be on one line.

Reply:
Thanks, fixed in 117f0bc.